modules/lib/src/main/resources/assets/js/app/AppWrapper.ts

```diff
@@ -149,7 +150,7 @@ export class AppWrapper
     }
 
     private updateUrl(widget: Widget): void {
-        if (widget.getUrl() === UrlAction.BROWSE.toString()) {
+        if (`${UrlHelper.getWidgetBaseUrl()}/${widget.getUrl()}` === UrlAction.BROWSE.toString()) {
```

Comment:
There are many places where you create a string `${UrlHelper.getWidgetBaseUrl()}/${widget.getUrl()}`

Maybe create a `UrlHelper.getWidgetUrl(widget)` method instead that would return `${UrlHelper.getWidgetBaseUrl()}/${widget.getUrl()}` ?

Reply:
I thought about it, we have to use it for  `iconUrl` as well. Actually, `widget.getUrl()` and `iconUrl` return a part of path, not url